### PR TITLE
Classnames returns 'undefined' - Add a little logic

### PR DESCRIPTION
**Pull request by sidonaldson**
no more undefined class names
#21

### CODE REVIEW

File: src/KeyboardButton.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default class KeyboardButton extends PureComponent {
 		return (
 			<button
 				type="button"
-				className={`keyboard-button ${this.props.classes}`}
+				className={`keyboard-button ${this.props.classes ? this.props.classes : ''}`}
```
Comment (WiaczeslawP) [inline src/KeyboardButton.js]: I would prefer to define a default value for this property

Comment (sidonaldson) [reply src/KeyboardButton.js]: @WiaczeslawP agreed, this is mostly just a fix, feel free to edit the PR to your liking. I normally use the`classnames` package but I think that's over kill in this case.

Could amend to 
```
let classes = this.props.classes || '';
render...
```

**Review by sidonaldson — COMMENTED** [review]
Thoughts?

**Comment by sidonaldson** [comment]
@WiaczeslawP any further thoughts on this PR?

My aim with these PR's was to help fix a few tiny issues and delicately add extra CSS definitions without forcing a major update or breaking change. I'm planning on using this in production and I'd rather contribute to your great work rather than fork it. :)

File: src/KeyboardButton.js [hunk 2]
```diff
@@ -17,10 +17,13 @@ export default class KeyboardButton extends PureComponent {
 	handleClick = () => this.props.onClick(this.props.value)
 
 	render() {
+	
+		let classes = this.props.classes || '';
```
Comment (WiaczeslawP) [inline src/KeyboardButton.js:21]: `classes` is never reassigned, it's better to use `const` instead

Comment (sidonaldson) [reply src/KeyboardButton.js:21]: @WiaczeslawP sh*t. This has introduced a breaking change. The latest build on NPM does not have the above line of code at all - completely omitted. ` classes is not defined`

Comment (WiaczeslawP) [reply src/KeyboardButton.js:21]: Oh. Fixed in version 1.3.10

**Pull request merged**